guides/source/threading_and_code_execution.md

```diff
@@ -320,5 +473,5 @@ interlock, which lock level they are holding or awaiting, and their current
 backtrace.
 
 Generally a deadlock will be caused by the interlock conflicting with some other
-external lock or blocking I/O call. Once you find it, you can wrap it with
-`permit_concurrent_loads`.
+external lock or blocking input/output call. Once you find it, you can wrap it
```

Comment:
"blocking I/O" is the term of art; if a reader doesn't know "I/O", they're certainly not going to know "blocking".